pylint/constants.py

```diff
@@ -176,6 +176,10 @@ class DeletedMessage(NamedTuple):
     ),
     # https://github.com/PyCQA/pylint/pull/3571
     DeletedMessage("C0330", "bad-continuation"),
+    # No PR
```

Comment:
How do you know that it was deleted then ?

Reply:
There is no mention of it anywhere in the codebase  but we do have a test that disables it. There are some messages about it online but searching for either the ID or name on GitHub doesn't show an issue or PR related to the removal.